l10n_it_fatturapa_out/views/account_view.xml

```diff
@@ -44,6 +44,24 @@
                         </group>
                     </group>
                 </page>
+                <page string="DDT Vuoti" attrs="{'invisible': [('electronic_invoice_subjected', '=', False)]}">
```

Comment:
Le stringhe sorgente andrebbero in inglese e poi tradotte su http://translation.odoo-community.org/ . Vedi anche https://github.com/OCA/l10n-italy/wiki/Traduzioni

Potresti però spiegare cosa intendi qui con `DDT vuoti`?

La funzionalità poi andrà spiegata anche in `readme/USAGE.rst`